pkg/pushsync/pushsync.go

```diff
@@ -319,10 +300,11 @@ func (ps *PushSync) handleDeliveryResponse(ctx context.Context, w protobuf.Write
 		return err
 	}
 
-	// since all PSS messages comes through push sync, deliver them here if this node is the destination
-	err = ps.deliverToPSS(ctx, chunk)
-	if err != nil {
-		ps.logger.Debugf("error pss delivery for chunk %v: %v", chunk.Address(), err)
+	if ps.deliveryCallback != nil {
+		// since every chunk received with pushsync will be
+		// tried to be unwrapped, we swallow the error here
+		_ = ps.deliveryCallback(ctx, chunk)
```

Comment:
Even this log line can be printed for a contend addressed chunk identified as a trojan chunk, I would keep the error log and remove if it appears too frequent later.

Reply:
but then it would constantly appear for all content addressed chunks that are unwrapped as garbage and will say that the topic is not found, which is annoying. so maybe we can have the logline, but remove the pss error which is returned when no handlers are found? i find this easier and better. if there's no handlers - too bad

Reply:
Ok, let's ignore ErrNoHandler but log other errors.

Reply:
It appears to create an import circle `github.com/ethersphere/bee/pkg/pss: import cycle not allowed`. :(

Reply:
hm. lovely :)

Reply:
hm. great. i don't know how to get around this. not returning this error in the first place seems like the cleanest thing to do.